app/editor/src/features/admin/actions/ActionFilter.tsx

```diff
@@ -2,17 +2,17 @@ import React from 'react';
 import { IconButton, Row, Text } from 'tno-core';
 
 interface IAdminFilterProps {
-  setGlobalFilter: (filterValue: any) => void;
+  onFilterChange: (value: string) => void;
 }
 
-export const TagListFilter: React.FC<IAdminFilterProps> = ({ setGlobalFilter }) => {
+export const ActionFilter: React.FC<IAdminFilterProps> = ({ onFilterChange }) => {
   const [filter, setFilter] = React.useState<string>('');
   return (
     <Row className="filter-bar" justifyContent="center">
       <Text
         onChange={(e) => {
           setFilter(e.target.value);
-          setGlobalFilter(e.target.value);
+          onFilterChange(e.target.value);
```

Comment:
I tried to move this logic to a `React.useEffect` but it results in an infinite loop after entering text in the search field.